src/views/scroll-editor/manuscript-toolbar.vue

```diff
@@ -337,6 +365,7 @@ export default class ManuscriptToolbar extends Vue {
 
     private selectedSide: string = 'left';
     private metricsInput: number = 1;
+    private ppi = Math.sqrt((Math.pow(window.screen.width, 2)) + (Math.pow(window.screen.height, 2)) ) / 15;
```

Comment:
You should add a comment here explaining why this works. And why aren't you using the ppi measure from the edition?